src/views/FactboxEditor/index.tsx

```diff
@@ -0,0 +1,295 @@
+import { useQuery, useCollaboration, useYValue } from '@/hooks'
+import { AwarenessDocument } from '@/components/AwarenessDocument'
+import { type ViewProps, type ViewMetadata } from '@/types/index'
+import { ViewHeader } from '@/components/View'
+import { BookTextIcon } from '@ttab/elephant-ui/icons'
+import type * as Y from 'yjs'
+import { Bold, Italic, Text, OrderedList, UnorderedList } from '@ttab/textbit-plugins'
+import Textbit, { DropMarker, Menu, type PluginRegistryAction, Toolbar, usePluginRegistry, useTextbit } from '@ttab/textbit'
+import { type HocuspocusProvider } from '@hocuspocus/provider'
+import { type AwarenessUserData } from '@/contexts/CollaborationProvider'
+import { type PropsWithChildren, useEffect, useMemo, useState } from 'react'
+import { withCursors, withYHistory, withYjs, YjsEditor } from '@slate-yjs/core'
+import { createEditor } from 'slate'
+import { type YXmlText } from 'node_modules/yjs/dist/src/internals'
+import { TextBox } from '@/components/ui'
+import { Button } from '@ttab/elephant-ui'
+import { createStateless, StatelessType } from '@/shared/stateless'
+import { useSession } from 'next-auth/react'
+import { createDocument } from '@/lib/createYItem'
+import { factboxDocumentTemplate } from '@/lib/templates/factboxDocumentTemplate'
+
+const meta: ViewMetadata = {
+  name: 'FactboxEditor',
```

Comment:
Raising the question whether this is the path we want to use, it doesn't really adhere to the rest plannings planning events event etc (unsure of what we call the Image-paths). Is factbox enough? It might be a better path.

factboxes and factbox?

Reply:
I'm for changing into a simpler format, as you suggest here.
Since they reside in the `templates`folder, simply calling them by their initial names (planning, event, factbox) should be enough.
I can add that to this PR if that's ok with you and @dannelundqvist ?

Reply:
Now I understand that it's the name of the view itself you were talking about.
So:
FactboxEditor -> Factbox
FactboxSearch -> Factboxes
Sounds good to me, easy enough to change.
Made yet another quick PR if this is accepted:
https://github.com/ttab/elephant-chrome/pull/689

Reply:
:heavy_check_mark: 
Lets skip the renaming of the templates (for now)